frappe/core/doctype/doc_history_temp/doc_history_temp.json

```diff
@@ -0,0 +1,152 @@
+{
```

Comment:
But does the user expect the changes to be shown instantly? Also is there a UI compontent to this?